.circleci/config.yml

```diff
@@ -264,6 +270,18 @@ jobs:
             - frontend/src/autogen/proto.js
             - lib/streamlit/proto
 
+  # The following jobs inherit from python-3.7.4. In a few cases, there are
+  # steps that are skipped based on the name of the current job (see, e.g.,
+  # "Run frontend tests").
+  python-3.8:
+    <<: *job-template
+    docker:
+      - image: circleci/python:3.8.1
+  python-3.5:
+    <<: *job-template
+    docker:
+      - image: circleci/python:3.5.9
+
   cypress:
     docker:
       - image: circleci/python:3.7.4-stretch
```

Comment:
Why keep Cypress at 3.7.4 but have jstest run at 3.8?

Reply:
I just updated the PR message to say this--I had updated the commit comment, and didn't realize it wouldn't carry over to the PR. :(

Cypress tests failed at 3.8 because of some font difference, while jstest are fine. Again, I updated that just to run against the latest and because I expect Python 3.7.4 testing to go away soon, while this is held back until we solve the Cypress/font issue that arose.